{"name": "test_3", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 > 21344242", "trace": "@allure.feature(\"模块二\")\n    @allure.story(\"这是一个很可爱的测试用例\")\n    def test_3():\n>       assert 0>21344242\nE       assert 0 > 21344242\n\n生成报告.py:19: AssertionError"}, "start": 1566180514160, "stop": 1566180514161, "uuid": "01166b4e-855e-4fb5-8937-c179c4c8c5da", "historyId": "928d8c5b60f16559de1d43d923cc8f2a", "fullName": "生成报告#test_3", "labels": [{"name": "story", "value": "这是一个很可爱的测试用例"}, {"name": "feature", "value": "模块二"}, {"name": "parentSuite"}, {"name": "suite", "value": "生成报告"}, {"name": "subSuite"}, {"name": "host", "value": "DESKTOP-3VGE1PR"}, {"name": "thread", "value": "12652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "生成报告"}]}